src/PlayFabBuddy.Lib/Interfaces/Adapter/IPlayStreamAdapter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using PlayFabBuddy.Lib.Adapter.Accounts;
+
+namespace PlayFabBuddy.Lib.Interfaces.Adapter;
+
+public interface IPlayStreamAdapter
+{
+    /// <summary>
+    /// All Players
+    /// </summary>
+    public const string AllPlayersSegmentName = "All Players";
+    
+    /// <summary>
+    /// Last login greater than 30 days ago
+    /// </summary>
+    public const string LapsedPlayersSegmentName = "Lapsed Players";
+    
+    /// <summary>
+    /// Total value to date in USD greater than $0.00
+    /// </summary>
+    public const string PayersSegmentName = "Payers";
+    
+    /// <summary>
+    /// First login less than 14 days ago AND First login greater than or equal to 7 days ago AND Last login less than 7 days ago
+    /// </summary>
+    public const string WeekTwoActivePlayersSegmentName = "Week Two Active Players";
+
+    /// <summary>
+    /// Get all players in a given segment, identified by <paramref name="segmentId"/>
+    /// </summary>
+    /// <param name="segmentId">The segment's ID</param>
+    /// <returns>All the Master Player Accounts in the segment</returns>
+    public Task<List<MasterPlayerAccountAdapter>> GetPlayersInSegment(string segmentId);
```

Comment:
Do we want to pass the "Adapter" along here or do we want to use the Entity as DTO instead? As far as i can see there is no operation performed on the entity that's managed here.

Reply:
I'd much rather use the Entity - I only had teh impression that usign teh Adapter was preferred

Reply:
Updated